ecommerce/frontend/src/components/UserReview.js

```diff
@@ -0,0 +1,128 @@
+
+    useEffect(() => {
+        // When current page is changed, makes a call to backend and gets the new reviews.
+        async function fetch() {
+            try {
+                setIsLoading(true)
+                const tmp = [
+                    {
+                        comment:
+                            'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.',
+                        author: 'Jeff Einstein',
+                        date: 'December 23',
+                        rating: 4,
+                    },
+                    {
+                        comment: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.',
+                        author: 'Hasan Kaya',
+                        date: 'December 11',
+                        rating: 3,
+                    },
+                    {
+                        comment:
+                            'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.',
+                        author: 'Jeff Einstein',
+                        date: 'December 23',
+                        rating: 4,
+                    },
+                    {
+                        comment: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.',
+                        author: 'Hasan Kaya',
+                        date: 'December 11',
+                        rating: 3,
+                    },
+                    {
+                        comment:
+                            'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.',
+                        author: 'Jeff Einstein',
+                        date: 'December 23',
+                        rating: 4,
+                    },
+                    {
+                        comment: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.',
+                        author: 'Hasan Kaya',
+                        date: 'December 11',
+                        rating: 3,
+                    },
+                ]
+                const urlStr = `/product/${productId}/review?pagesize=${pageSize}&currentpage=${currentPage}`
+                console.log(urlStr)
```

Comment:
for now completely fine since we don't know the endpoint's param, just leaving this here as a reminder for later to check the exact names of the params.